app-emulation/virtio-win/virtio-win-0.1.160.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Please use 7.

Otherwise lgtm. I'm traveling and can't merge, but if someone else wants to, go for it.

Reply:
It is done ;-)